src/shared/languageServer/utils/validator.ts

```diff
@@ -0,0 +1,38 @@
+/*!
```

Comment:
`src/shared/languageServer/utils/validator.ts` is a deep, verbose path. Can be renamed to `src/shared/lsp/validator.ts`. We aren't likely to have tens of modules in there, so "utils/" is unnecessary nesting.